### PR TITLE
Add ForceDisableShadowInit

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: tensilelite/Tensile/Common.py [hunk 1]
```diff
@@ -1028,6 +1028,9 @@ def getArchitectureName(gfxName):
 
     # in mix mode F8 need to convert to F16, do this before(0) ds or after(1) ds
     "ConvertAfterDS":              [False, True],
+
+    # Force disable shadow init to release more sgpr in preloop
+    "ForceDisableShadowInit":      [False, True],
```
Comment (jichangjichang) [inline tensilelite/Tensile/Common.py:1033]: Suggest not to add new kernel argument for just shawdoinit setup

**Comment by Jinp800125** [comment]
some case test might not pass.

File: tensilelite/Tensile/Common.py (hunk 1 above)
Comment (KKyang) [reply tensilelite/Tensile/Common.py:1033]: It's a must because lack of sgprs, or we can remove this feature completely.

**Comment by KKyang** [comment]
> some case test might not pass.

Fixed

File: tensilelite/Tensile/KernelWriter.py [hunk 2]
```diff
@@ -2068,85 +2071,8 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
...
-    lralwaMod.addComment0("init: add agpr [%u...%u) to pool" % \
-                         (0, numAccvgprs))
-
-    lralwaMod.addComment2("Local Read Addresses")
-
-    # tile assignments
-    lralwaMod.addComment1("local read addresses: tile assignments a/b")
-    lralwaMod.add(self.lraTileAssignment(kernel, tensorParametersA, tensorParametersB))
-
-    tPM = tensorParametersA["tpsMetadata"] if tensorParametersA["is_sparse"] else tensorParametersB["tpsMetadata"]
```
Comment (hcman2) [inline tensilelite/Tensile/KernelWriter.py]: you remove tPM here which will be used in the following code segment:
localReadCodeM, packCodeM = self.localReadDo(kernel, plrIdx*self.states.numIterPerCoalescedReadMetadata, iui*self.states.numReadsIterCoalescedMetadata, espi, tPM)
in line 2148.

Comment (KKyang) [reply tensilelite/Tensile/KernelWriter.py]: fixed

**Review by hcman2 — APPROVED** [review]
ok if CI pass.

**Review by hcman2 — APPROVED** [review]
You may add more coverage in test yaml.

File: tensilelite/Tensile/Common.py (hunk 1 above)
Comment (hcman2) [reply tensilelite/Tensile/Common.py:1033]: how much performance difference if we force this on?

Comment (KKyang) [reply tensilelite/Tensile/Common.py:1033]: Can't generate some kernels without this flag.

**Review by Jinp800125 — APPROVED** [review]

**Pull request merged**